gradio/external.py

```diff
@@ -429,6 +460,10 @@ def streamline_config(config: dict) -> dict:
     config = streamline_config(config)
     api_url = "https://hf.space/embed/{}/api/predict/".format(model_name)
     headers = {"Content-Type": "application/json"}
+    ws_url = "wss://spaces.huggingface.tech/{}/queue/join".format(model_name)
+
+    ws_fn = get_ws_fn(ws_url)
+    connect_to_websocket = use_websocket(config)
 
     fns = []
     for d, dependency in enumerate(config["dependencies"]):
```

Comment:
Also consider whether the specific function itself enables queue or not (see my comment to the PR)